tpcx_bb/xbb_tools/utils.py

```diff
@@ -359,60 +363,10 @@ def get_tpcxbb_argparser_commandline_args():
     parser = argparse.ArgumentParser(description="Run TPCx-BB query")
     print("Using default arguments")
     parser.add_argument(
-        "--data_dir",
-        default="/datasets/tpcx-bb/sf1000/new_parquet/",
-        type=str,
-        help="Data Dir",
-    )
-    parser.add_argument(
-        "--file_format", default="parquet", type=str, help="File format"
-    )
-    parser.add_argument(
-        "--output_dir",
-        default="./",
+        "--config_file",
```

Comment:
There is some tight coupling between utility/query logic and the CLI arguments. We'll need to set defaults for these arguments in the yaml if we remove them from the CLI

Reply:
I think I've addressed this now, but would appreciate a second look